newIDE/app/src/SceneEditor/index.js

```diff
@@ -1454,7 +1488,9 @@ export default class SceneEditor extends React.Component<Props, State> {
                 }
                 onObjectCreated={this._onObjectCreated}
                 onObjectSelected={this._onObjectSelected}
-                onRenameObject={this._onRenameObject}
+                renamedObjectWithContext={this.state.renamedObjectWithContext}
+                onRenameStart={this._onRenameStart}
+                onRenameFinish={this._onRenameFinish}
```

Comment:
Nit, but important to be extra clear and consistent with `onObjectCreated`, `onEditrObject`...:
```suggestion
                onRenameObjectStart={this._onRenameObjectStart}
                onRenameObjectFinish={this._onRenameObjectFinish}
```